### PR TITLE
Add template for security-check

### PR DESCRIPTION
**Pull request by phavekes**

### CODE REVIEW

File: workflow-templates/daily-security-check.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+name: Daily security check
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+  push:
```
Comment (MKodde) [inline workflow-templates/daily-security-check.yml]: Is this intentional, to also run the GHA on `push`? If so, maybe update the GHA template description to include this fact?

File: workflow-templates/daily-security-check.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        uses: actions/checkout@v2
+
+      # PHP checks
+      - name: Check for php composer project
+        id: check_composer
+        uses: andstor/file-existence-action@v2
+        with:
+          files: "composer.lock"
+      - name: Run php local security checker
+        if: steps.check_composer.outputs.files_exists == 'true'
```
Comment (MKodde) [inline workflow-templates/daily-security-check.yml:23]: Nifty!

File: workflow-templates/daily-security-check.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          files: "yarn.lock"
+      - name: Setup node
+        if: steps.check_node.outputs.files_exists == 'true'
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      - name: Yarn Audit
+        if: steps.check_node.outputs.files_exists == 'true'
+        run: yarn audit --level high --groups dependencies optionalDependencies
+        continue-on-error: true
```
Comment (MKodde) [inline workflow-templates/daily-security-check.yml]: Usually we use Yarn for getting hold of Node packages. However, in EngineBlock we use NPM for this task. If you want to make this action ultra portable, you might want to add a second check on a `package.lock` file. And in that case perform an `npm audit`

File: workflow-templates/daily-security-check.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+      - name: Check for java maven project
+        id: check_maven
+        uses: andstor/file-existence-action@v2
+        with:
+          files: "pom.xml"
+      - name: Setup java if needed
+        if: steps.check_maven.outputs.files_exists == 'true'
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```
Comment (MKodde) [inline workflow-templates/daily-security-check.yml:80]: Aren't the Java guys moving to Java 13?

**Review by MKodde — COMMENTED** [review]
Looking great! See some nitty witty feedback below.

I'm going to test-drive this in Stepup-WebAuthn for sure.

File: workflow-templates/daily-security-check.yml (hunk 1 above)
Comment (phavekes) [reply workflow-templates/daily-security-check.yml]: No, it should not run on push. I'll change that

File: workflow-templates/daily-security-check.yml (hunk 3 above)
Comment (phavekes) [reply workflow-templates/daily-security-check.yml]: I've added an (untested) npm audit block.

File: workflow-templates/daily-security-check.yml (hunk 4 above)
Comment (phavekes) [reply workflow-templates/daily-security-check.yml:80]: We'll move to 13 when they get there

File: workflow-templates/daily-security-check.yml (hunk 3 above)
Comment (MKodde) [reply workflow-templates/daily-security-check.yml]: When we implement this action in EB, we will eat that pudding.

File: workflow-templates/daily-security-check.properties.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Daily security checks",
+    "description": "Use this template to create a GitHub Actions workflow to run daily checks on outdated versions. supports PHP, Node, Pyhton and JAVA projects",
```
Comment (tvdijen) [inline workflow-templates/daily-security-check.properties.json]: Typo `Pyhton`

**Review by MKodde — APPROVED** [review]

**Pull request merged**